### PR TITLE
Empty page with sidebar and header

### PR DESCRIPTION
**Pull request by oyinmi**
Modify and arranged the sidebar, created and added the header and created empty page to create new board

### CODE REVIEW

File: client/src/components/Header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                        height={30}
+                        alt="icon"
+                    />
+                </div>
+                <h1 className={styles.headingText}>kanban</h1>
+            </div>
+
+            <div className={styles.subheader}>
+                <h2>Platform Launch</h2>
+                <p>+ Add New Task</p>
```
Comment (Erikang20) [inline client/src/components/Header/index.jsx:25]: It seems that this element was already created. And exists here: https://github.com/Erikang20/Kanban-task-management-web-app/blob/main/client/src/components/Header/addTaskBtn.jsx So I think we can get rid of this :)

File: client/src/components/Header/index.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	return (
+		<div className={styles.header}>
+            <div className={styles.heading}>
+                <div className={styles.logo}>
+                    <Image
+                        src={logoLight}
+                        width={150}
+                        height={30}
+                        alt="icon"
+                    />
```
Comment (Erikang20) [inline client/src/components/Header/index.jsx:18]: The logo also exists already here https://github.com/Erikang20/Kanban-task-management-web-app/blob/main/client/src/components/Header/logo.tsx

File: client/src/components/Header/index.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                </div>
+                <h1 className={styles.headingText}>kanban</h1>
+            </div>
+
+            <div className={styles.subheader}>
+                <h2>Platform Launch</h2>
```
Comment (Erikang20) [inline client/src/components/Header/index.jsx:24]: I'd add this element here: https://github.com/Erikang20/Kanban-task-management-web-app/blob/main/client/src/components/Header/header.jsx#L11 and style it accordingly

File: client/src/components/Header/styles.module.scss
Comment (Erikang20) [inline client/src/components/Header/styles.module.scss:1]: We have styles for the header here: https://github.com/Erikang20/Kanban-task-management-web-app/blob/main/client/src/components/Header/styles.module.scss so I'd add whatever you need in that file :)

File: client/src/components/Sidebar/index.jsx [hunk 4]
```diff
@@ -23,27 +25,77 @@ const Sidebar = () => {
...
+								width={150}
+								height={30}
 								alt="icon"
 							/>
-							Kanban
-						</h1>
-					</div>
+						</div>
+						<h1 className={styles.headingText}>kanban</h1>
+					</div> */}
```
Comment (Erikang20) [inline client/src/components/Sidebar/index.jsx:44]: Unnecessary comments?

File: client/src/components/Sidebar/index.jsx [hunk 5]
```diff
@@ -23,27 +25,77 @@ const Sidebar = () => {
...
+								className={styles.newBoardLinkIcon}
+								src={fluentBoard2}
+								width={30}
+								height={15}
+								alt="icon"
+							/>
+							<p className={styles.newBoardLink}>+ Create New Board</p>
+						</a>
+						
+
```
Comment (Erikang20) [inline client/src/components/Sidebar/index.jsx:95]: Apologies, it seems that we have the create new board link existing here: https://github.com/Erikang20/Kanban-task-management-web-app/blob/main/client/src/components/Sidebar/index.jsx#L51

File: client/src/components/Sidebar/styles.module.scss [hunk 6]
```diff
@@ -83,4 +89,108 @@
...
+	.newBoardLink {
+		font-weight: 700;
+		font-size: small;
+		color: #635fc7;
+	}
+
+	// .hidesidebar {
+	// 	width: 20vh;
+	// 	padding-left: 20px;
+	// }
```
Comment (Erikang20) [inline client/src/components/Sidebar/styles.module.scss:195]: unnecessary comments?

**Review by Erikang20 — CHANGES_REQUESTED** [review]
It seems that some things need to be removed from this PR as they already exist in the code. Please let me know if you need some help :)

File: client/src/components/Header/index.jsx (hunk 1 above)
Comment (oyinmi) [reply client/src/components/Header/index.jsx:25]: Do you mean to get rid of the whole subheader or just the button?

File: client/src/components/Header/index.jsx (hunk 2 above)
Comment (oyinmi) [reply client/src/components/Header/index.jsx:18]: It wasn't available at the time I pulled and started working on the task. But you can replace it if that's okay.

File: client/src/components/Header/index.jsx (hunk 3 above)
Comment (oyinmi) [reply client/src/components/Header/index.jsx:24]: Alright. That's fine.

File: client/src/components/Sidebar/index.jsx (hunk 5 above)
Comment (oyinmi) [reply client/src/components/Sidebar/index.jsx:95]: This task wasn't created when I picked my task to work on It.

File: client/src/components/Header/index.jsx (hunk 2 above)
Comment (Erikang20) [reply client/src/components/Header/index.jsx:18]: yeah this is probably my fault. These are some changes I made that I mentioned yesterday that I needed to complete my last task.  What I'd do is to rebase against main and run the app and see what elements are not longer needed  ;) Let me know if you need help with rebase.

**Pull request closed without merging**